src/components/AddPaymentMethodMenu.tsx

```diff
@@ -108,6 +109,7 @@ function AddPaymentMethodMenu({
                               text: translate('common.personalBankAccount'),
                               icon: Expensicons.Bank,
                               onSelected: () => {
+                                  IOU.completePaymentOnboarding(CONST.IOU.PAYMENT_SELECTED.PBA);
```

Comment:
Should we add completePaymentOnboarding to 

https://github.com/Expensify/App/blob/4fe86eb6baf6d71aa8757b2362a788d20ee42109/src/components/AddPaymentMethodMenu.tsx#L83

Reply:
I can try moving it inside the function itself, sure.

Reply:
AddPaymentMethodMenu is implemented in two separate components so I'd have to add the same code twice. How about I create a wrapper function such as this:

```
const onPaymentItemSelected = (paymentMethod: PaymentMethod) => {
    IOU.completePaymentOnboarding(paymentMethod);
    onItemSelected(paymentMethod);
}
```

And replace the part you've selected with `onPaymentItemSelected(paymentMethod) `?

Reply:
You mean we should create a new wrapper function called onPaymentItemSelected and use it in AddPaymentMethodMenu. It's fine to me

Reply:
I'll do that but it won't change the functionality, just move the code around. You can test it in the meantime.